shinysdr/plugins/limesdr.py

```diff
@@ -0,0 +1,306 @@
+# Copyright 2013, 2014, 2015, 2016, 2017 Kevin Reid <kpreid@switchb.org>
```

Comment:
This should have 2018 and your name; or also the preceding dates and both of our names (break that to the next line) if you feel that you copied a substantial portion from my code.